### ISSUE TITLE
deskdock pro is not showing cursor on mobile screen after first launch.

### ISSUE THREAD

**Issue by asfgpr** [Genymobile/scrcpy#2208]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [e.g. Debian, Windows, macOS...]
 - scrcpy version: [e.g. 1.12.1]
 - installation method: [e.g. manual build, apt, snap, brew, Windows release...]
 - device model:
 - Android version: [e.g. 10]

**Describe the bug**

On errors, please provide the output of the console (and `adb logcat` if relevant).

```
Please paste terminal output in a code block.
```

Please do not post screenshots of your terminal, just post the content as text instead.

**Comment by asfgpr** [comment]
when i open deskdock app first time then its working fine and when i disconnect and connect again then i the cursor is not showing on mobile screen but working fine all other feature

can anyone solve this issue in deskdock app

**Comment by gaussandhisgun** [comment]
this is wrong repo, this is scrcpy, which is completely unrelated. deskdock is not supported too. and, also, you provide no information about your setup, so even if there occasionally are any people who theoretically could help, they cant because of no info.

**Comment by jovyllebermudez** [comment]
Same got issue. But yeah, this is scrpy

**Comment by Yuki-J1** [comment]
> 当我第一次打开DeskDock应用程序时，它工作正常，当我断开连接并再次连接时，光标未显示在移动屏幕上，但所有其他功能都可以正常工作
> 
> 任何人都可以在桌面扩展坞应用程序中解决此问题吗

https://github.com/debauchee/barrier/issues/189#issuecomment-901331388
